test/functional/services/saved_objects_finder.ts

```diff
@@ -32,7 +32,7 @@ export class SavedObjectsFinderService extends FtrService {
     for (let i = 0; i < listItems.length; i++) {
       const listItem = await listItems[i].findByClassName('euiSelectableListItem__text');
       const text = await listItem.getVisibleText();
-      if (text.includes(type)) {
+      if (text === type) {
```

Comment:
This was causing a failure when the type parameter was `Visualization` because it was matching `Lens Visualization` instead. This test failure did not occur before this PR because the list was previously not sorted.